modules/fonts/sources/google-fonts.sh

```diff
@@ -8,36 +8,35 @@ echo "Installation of google-fonts started"
 rm -rf "${DEST}"
 
 for FONT in "${FONTS[@]}"; do
-    FONT="${FONT#"${FONT%%[![:space:]]*}"}" # Trim leading whitespace
-    FONT="${FONT%"${FONT##*[![:space:]]}"}" # Trim trailing whitespace
-    mkdir -p "${DEST}/${FONT}"
+    if [ -n "$FONT" ]; then
+        FONT="${FONT#"${FONT%%[![:space:]]*}"}" # Trim leading whitespace
+        FONT="${FONT%"${FONT##*[![:space:]]}"}" # Trim trailing whitespace
+        mkdir -p "${DEST}/${FONT}"
 
-    readarray -t FILE_REFS < <(
-        if JSON=$(
-            curl -s "https://fonts.google.com/download/list?family=${FONT// /%20}" | # spaces are replaced with %20 for the URL
-            tail -n +2 # remove first line, which as of March 2024 contains ")]}'" and breaks JSON parsing
```

Comment:
It's hard for me to see the changes you made from the diff alone, but please incorporate the changes from my latest commit to your PR.

Reply:
@fiftydinar you're too fast smh!!!

Reply:
Did you even read the diff?

Reply:
> Did you even read the diff?

Didn't notice any issues, what did I miss?

Reply:
I did not notice any changes, except reverting my previous commit.

Reply:
As far as I can see in diff, your commits are not reverted.
Only some if conditions are added around them & echos for better error handling.

Reply:
This one is hard to tell, but really the only diff on this one is `if [ -n "$FONT" ]; then` and then adding a layer of indentation for the rest